src/solvers/nlsolve.jl

```diff
@@ -87,5 +87,6 @@ function only_fj!(fj!::T) where T
     end
 
     _jv!(x) = nothing
-    return NLSolvers.VectorObjective(f!,j!,fj!,jv!) |> NEqProblem
-end
\ No newline at end of file
+    return NLSolvers.VectorObjective(_f!,_j!,_fj!,_jv!) |> NEqProblem
```

Comment:
I change this line to the function definitions with underscore (_f!, _j!, ...), but I still can't make it to work with Solvers.nlsolve 🥲

Reply:
it was a bug the constructor, fixed

Reply:
there was one more problem down the line haha, `NLSolvers.OptimizationProblem` uses a kwarg-only API, and i was using a positional one. i can run `psat` locally with all it's combinations on the latest commit

Reply:
Yes, it is running for me too but apparently when just giving initial guesses for the volumes the solver is not converging e.e